in/install.sh.in

```diff
@@ -20,6 +20,7 @@ main() {
 
     sed "s|%PTC_BIN|${LIB_DIR}/push-to-cloud.jar|" \
         in/unit/push-to-cloud.service.in > ${UNIT_DIR}/push-to-cloud.service \
+        && sed -i '/User=ptc/d' ${UNIT_DIR}/push-to-cloud.service \
         && systemctl --user daemon-reload
```

Comment:
If this is no longer a user daemon, please remove the `--user`

Reply:
My goal was to allow this to still be run as a user level daemon if installed through this install script.  If installed through the rpm then it would be a system level daemon.  I can make this install script act similar to the rpm but then it will require root for the install and upkeep.  Let me know which you would prefer.